lib/library.h

```diff
@@ -179,7 +179,9 @@ class CPPCHECKLIB Library {
     bool isNotLibraryFunction(const Token *ftok) const;
     bool matchArguments(const Token *ftok, const std::string &functionName) const;
 
+    enum RetvalErrorType { DEFAULT, ERROR_CODE };
     bool isUseRetVal(const Token* ftok) const;
```

Comment:
I wonder if we want two separate functions. We could add a enum constant `NONE`.